### PR TITLE
net-im/prosody: replace fixed lib64 by getlib_dir

### PR DESCRIPTION
**Pull request by xdch47**
- this fixes issue for installed module on 32bit platforms

Package-Manager: Portage-2.3.49, Repoman-2.3.10

### CODE REVIEW

**Comment by klausman** [comment]
I approve of this PR in principle, but I am not authorized to merge it.

File: net-im/prosody/prosody-0.10.2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic multilib systemd
```
Comment (Zlogene) [inline net-im/prosody/prosody-0.10.2-r1.ebuild]: you do not need multilib

Comment (xdch47) [reply net-im/prosody/prosody-0.10.2-r1.ebuild]: well, i didn't add multilib just adopt it from the previous versions to force an installation to lib64… not sure, if it's a regression when we remove it.

Comment (a17r) [reply net-im/prosody/prosody-0.10.2-r1.ebuild]: @xdch47 it's not, because all you need is provided by >=EAPI-6.

Comment (xdch47) [reply net-im/prosody/prosody-0.10.2-r1.ebuild]: hopefully fixed (including some more cleanups + EPREFIX changes)

**Pull request closed without merging**